### ISSUE TITLE
[FEATURE] Show Calendar events on At a Glance

### ISSUE THREAD

**Issue by Americapax** [Deletescape-Media/Lawnchair#4098]
### Describe the feature

Hi, 
I would like to have the next calendar event in the At Glance widget on the homescreen (like the Pixel Launcher)

### Why would it be useful to add?

it would be useful to add it, so that the users may see their calendar event without opening their calendar app 

### Did the feature exist in Lawnchair v2? (Play Store version)

No

### Example(s)

![at-a-glance-pixel-launcher-why-youre-seeing-this](https://github.com/LawnchairLauncher/lawnchair/assets/142015783/883900b7-e770-4e9b-b8ec-d74b1b867693)

### Additional context

On the At a Glance widget, it is very useful (atleast for me)

(sorry for my bad English grammar, i am Danish, so English is not my first language)

**Comment by sarojpanda44** [comment]
Hahaha after reading the additional context I read your username 🤣😜 sorry no offence but it is hilarious

**Comment by RWCoding58** [comment]
I believe this should be possible, since if I'm not wrong Lawnchair's At A Glance is a stripped down version of Smartspacer which originally has an in-built calendar target.

*Recommendation:* This is a clear, desirable feature request that fits the project vision.

**Comment by SalvatoreNoschese** [comment]
Indeed. ATM I use smartspacer (I can disable flashlight, see a lot of extra details). Lawnchair at a glance ATM is very poor.

**Comment by michaelhofer** [comment]
That would be extremely useful =)

**Comment by DavisPaulus** [comment]
> That would be extremely useful =)

I second that
